### DISCUSSION TITLE
Poorly formatted math in revealjs on Chrome

### DISCUSSION THREAD

**Discussion by garthtarr** [quarto-dev/quarto-cli#3127] [Q&A]
Something I've noticed recently is that the MathJax equations in revealjs presentations are rendering in an ugly way on Chrome (at least on a Mac).

Specific example: https://quarto.org/docs/presentations/revealjs/demo/index.html#/latex-equations 

Chrome with default settings: 

<img width="1166" alt="Screenshot 2022-11-01 at 3 04 36 pm" src="https://user-images.githubusercontent.com/2189134/199156331-2ca89238-01b2-4a45-acc6-da39b5358fde.png">

Firefox with default settings:

![Screenshot 2022-11-01 at 3 04 45 pm](https://user-images.githubusercontent.com/2189134/199156405-9cee0ea2-87c2-442d-9209-cebf6f8e17f9.png)

I can right click and change the renderer to Common HTML and it looks OK, but the default looks terrible. 

Does anyone have suggestions about how I can change the default behaviour. Interestingly it doesn't seem to be a problem when I render to HTML format - I get a different set of options when right clicking, so it might be something to do with revealjs?

Options when using the revealjs format:
<img width="562" alt="Screenshot 2022-11-01 at 3 06 15 pm" src="https://user-images.githubusercontent.com/2189134/199156702-4253135d-20e9-493d-8669-7140ed1c740f.png">

 
Options when rendering to HTML format:
<img width="734" alt="Screenshot 2022-11-01 at 3 07 41 pm" src="https://user-images.githubusercontent.com/2189134/199156709-5430ae79-7e9a-4716-9e89-9bc9fa105073.png">

**Comment by mcanouil** [comment]
This seems to be a MathJax issue and does not seem to be related in any ways to Quarto.
It would probably be better to report this to MathJax rather than here where it cannot be solved.

**Reply by garthtarr** [reply]
Thanks for the reply, in the meantime while this is being fixed upstream, is there any way to specify a default MathJax renderer in the quarto YAML?

**Reply by garthtarr** [reply]
I was playing with this, and found that specifying MathJax directly "fixes" the problem by using MathJax v3.2.2

```
---
title: "Untitled"
format: 
  revealjs:
    html-math-method: mathjax
---

## Quarto

$y = mx+b$
```

Looks like this (totally fine):
<img width="1000" alt="Screenshot 2022-11-02 at 9 44 12 am" src="https://user-images.githubusercontent.com/2189134/199356065-27ad9d73-11a2-4f2a-9c15-c088056646db.png">




But when we don't explicitly specify mathjax, it loads MathJax v2.7.0 and looks like this:

```
---
title: "Untitled"
format: revealjs
---

## Quarto

$y = mx+b$
```

It looks like this:

<img width="1000" alt="Screenshot 2022-11-02 at 9 44 00 am" src="https://user-images.githubusercontent.com/2189134/199356083-7d2e7e54-1f56-416d-a425-aceb834479df.png">

**Reply by mcanouil** [reply]
Probably, by including some scripts configuration in your document header (`include-in-header`).
You'll have to look for what exactly in MathJax documentation
website.

**Comment by rressler** [comment]
Thanks! specifying MathJax directly "fixed" the problem for me of all my \hat{} being off to the right in Chrome but fine in Firefox.

**Reply by jtr13** [reply]
Exactly the issue I was having with \hat ... thanks!

**Comment by jtr13** [comment]
More on this here: https://github.com/quarto-dev/quarto-cli/discussions/4916

**Comment by ajfriend** [comment]
I just ran into this issue, and the `html-math-method: mathjax` worked for me. Thanks!
